esmvaltool/cmorizers/obs/cmorize_obs_eppley_vgpm_modis.py

```diff
@@ -0,0 +1,141 @@
+# pylint: disable=invalid-name
+"""ESMValTool CMORizer for Eppley-VGPM-MODIS data from Oregon State University.
+
+Tier
+
+Source
+   http://orca.science.oregonstate.edu/data/1x2/monthly/eppley.r2018.m.chl.m.sst/hdf
+
+Last access
+   20190515
+
+Download and processing instructions
+   Download and unpack all files under a single directory
```

Comment:
Maybe specify that only the `.gz` files should be downloaded (there are also `.tar` files with apparently the same content).

Reply:
@mattiarighi I add a note to download the annual `tar` files, which are the collection of the individual monthly gzipped files, so one has to deal with fewer downloads.